### PR TITLE
[Hub Generated] Review request for Microsoft.ApiManagement to add version stable/2019-01-01

### PR DESCRIPTION
**Pull request by solankisamir**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/solankisamir/azure-rest-api-specs/apim_nitfixes...master).

Changes to improve the Client experience on .NET client.

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-01-01/apimdeployment.json [hunk 1]
```diff
@@ -1079,7 +1079,7 @@
         },
         "enableClientCertificate": {
           "type": "boolean",
-          "description": "Property only meant to be used for Consumption SKU Service. This enforces a client certificate to be presented on each request to the gateway and enabled ability to authenticate the certificate in the policy.",
+          "description": "Property only meant to be used for Consumption SKU Service. This enforces a client certificate to be presented on each request to the gateway. This also enable the ability to authenticate the certificate in the policy on the gateway.",
```
Comment (nschonni) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-01-01/apimdeployment.json]:
```suggestion
          "description": "Property only meant to be used for Consumption SKU Service. This enforces a client certificate to be presented on each request to the gateway. This also enables the ability to authenticate the certificate in the policy on the gateway.",
```

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-01-01/definitions.json [hunk 2]
```diff
@@ -1634,7 +1691,17 @@
       ],
       "description": "Parameters supplied to the CreateOrUpdate certificate operation."
     },
-    "DeployConfigurationParameters": {
+    "DeployConfigurationParameters" :{
```
Comment (nschonni) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-01-01/definitions.json]:
```suggestion
    "DeployConfigurationParameters": {
```

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-01-01/definitions.json [hunk 3]
```diff
@@ -4098,7 +4165,17 @@
       },
       "description": "Sampling settings for Diagnostic."
     },
-    "SaveConfigurationParameter": {
+    "SaveConfigurationParameter" : {
```
Comment (nschonni) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-01-01/definitions.json]:
```suggestion
    "SaveConfigurationParameter":  {
```

**Comment by solankisamir** [comment]
@sergey-shandar can you take a look?

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**